common/src/types/v0/store/node.rs

```diff
@@ -1,31 +1,104 @@
 //! Definition of node types that can be saved to the persistent store.
 
-use crate::types::v0::{
-    openapi::models,
-    store::{
-        definitions::{ObjectKey, StorableObject, StorableObjectType},
-        ResourceUuid,
+use crate::{
+    types::v0::{
+        openapi::models,
+        store::{
+            definitions::{ObjectKey, StorableObject, StorableObjectType},
+            ResourceUuid,
+        },
+        transport::{self, NodeId},
     },
-    transport::{self, NodeId},
+    IntoOption,
 };
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 
+/// Generic labels associated with the node
 pub type NodeLabels = HashMap<String, String>;
 
+/// Data relating to the cordoning of a node
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+pub struct CordonedState {
+    /// labels used to enforce a node cordon
+    pub cordonlabels: Vec<String>,
+}
+
+impl CordonedState {
+    pub fn new(cordonlabels: Vec<String>) -> Self {
```

Comment:
missing doc comments

Reply:
fixed

Reply:
Did you forget to push? Still can't see them.